homeassistant/components/sky_hub/__init__.py

```diff
@@ -1 +1,20 @@
 """The sky_hub component."""
+from pyskyqhub.skyq_hub import SkyQHub
+
+from homeassistant.components.device_tracker import DOMAIN
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+PLATFORMS = ["device_tracker"]
+
+
+async def async_setup(hass, config):
```

Comment:
We don't need this at the moment, unless we're planning to make a breaking change and move configuration to the sky_hub domain section in the configuration.yaml. This integration is still just one platform.

Reply:
Without it the integration doesn't show in Configuration/Info

Reply:
Without it the integration does not appear in Configuration/Info

Reply:
What is "Configuration/Info"?

Reply:
The Configuration panel in HA has an Info tab. Integrations are listed there.

<img width="777" alt="Screenshot 2020-07-22 at 12 34 32" src="https://user-images.githubusercontent.com/7115783/88171742-c46ccc00-cc17-11ea-9381-218a15d8c5e6.png">

Reply:
That's just a frontend representation of the active integrations. It probably doesn't take pure platforms in to account.

We don't need to consider this in the backend. We should just follow the backend architecture. This integration is legacy so until it's refactored to follow our current design we shouldn't add code here.

Reply:
I could just return True which will setup the platform and then it will appear in this Tab, rather than validating hub is available.

Reply:
Just `return True` is ok.